### PR TITLE
Add README.md for each packages

### LINKED ISSUE #87: Revise READMEs
- Fix the root README.
  - Write about packages
  - Revise example code

- Add README.md in package folder

### PR DESCRIPTION
**Pull request by lukasjhan**
It closes #87

This PR contains

- meta tags for homepages
- README.md for each packages
- Revise badges and descriptions in README
- Fix typo broswer-crypto => browser-crypto

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-![Coverage](https://img.shields.io/badge/coverage-93%25-green)
+![Coverage](https://img.shields.io/codecov/c/github/openwallet-foundation-labs/sd-jwt-js)
```
Comment (lukasjhan) [inline README.md:1]: I added codecov project :)

File: docs/index.html [hunk 2]
```diff
@@ -3,7 +3,18 @@
...
+    <meta name="author" content="OpenWallet Foundation" />
+    <meta property="og:title" content="SD JWT - TypeScript JWT Library" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://sdjwt.js.org" />
+    <meta property="og:image" content="https://avatars.githubusercontent.com/u/114502864?s=200&v=4" />
+    <meta property="og:description" content="SD JWT is a TypeScript library implementing the IETF SD-JWT specification for selective disclosure of JWT claims, enhancing privacy and security in digital communications." />
+    <meta property="og:site_name" content="SD JWT - TypeScript JWT Library" />
+    <meta property="og:locale" content="en_US" />
+    <link rel="canonical" href="https://sdjwt.js.org" />
+    <title>SD JWT - TypeScript SD JWT Library</title>
```
Comment (lukasjhan) [inline docs/index.html:17]: I added meta tags for SEO. Hopefully it gets better

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 ![License](https://img.shields.io/github/license/openwallet-foundation-labs/sd-jwt-js.svg)
-![NPM](https://img.shields.io/npm/v/%40hopae%2Fsd-jwt)
+![NPM](https://img.shields.io/npm/v/%40sd-jwt%2Fcore)
```
Comment (cre8) [inline README.md:3]: we should keep in mind that we have multiple packages so theoretically it's wrong to just point to one version.

One the other hand we keep all the versions in sync. Do you think we should mention this somewhere?

File: packages/browser-crypto/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Browser Crypto support for SD JWT
+
+Check the detail description in our github [repo](https://github.com/openwallet-foundation-labs/sd-jwt-js).
+
+### Installation
+
+To install this project, run the following command:
+
+```bash
+npm install @sd-jwt/crypto-browser
```
Comment (cre8) [inline packages/browser-crypto/README.md:22]: should we mention in all the readme that they also could use yarn or pnpm?

File: packages/utils/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+![Coverage](https://img.shields.io/codecov/c/github/openwallet-foundation-labs/sd-jwt-js)
```
Comment (cre8) [inline packages/utils/README.md:1]: we should keep in mind that our coverage rate is calculated for the whole project, not for every library by itself.

I don't see the need to present it for each individual, just to mention it ;)

**Review by cre8 — APPROVED** [review]
just small notes, but all in all it looks good.

File: packages/utils/README.md (hunk 5 above)
Comment (lukasjhan) [reply packages/utils/README.md:1]: Yeah, good point. Maybe we should put the [codecov link](https://app.codecov.io/gh/openwallet-foundation-labs/sd-jwt-js) of our project. To see the details
What do you think about it?

File: packages/browser-crypto/README.md (hunk 4 above)
Comment (lukasjhan) [reply packages/browser-crypto/README.md:22]: I'll add yarn and pnpm :)

**Comment by lukasjhan** [comment]
@cre8 I updated some changes. :)

File: README.md [hunk 6]
```diff
@@ -38,63 +38,24 @@ By adhering to these design principles, "Selective Disclosure for JWT" aims to s
...
+## Version
 
-// Issuer issues a signed JWT credential with the specified claims and disclosure frame
-// returns an encoded JWT
-const credential = await sdjwt.issue(claims, privateKey, disclosureFrame);
+We keep all the versions of our packages in sync.
 
-// Holder may validate the credential from the issuer
-const valid = await sdjwt.validate(credential, publicKey);
+It means for example, that if you are using `@sd-jwt/core@1.0.0`, you should use `@sd-jwt/decode@1.0.0`, `@sd-jwt/present@1.0.0` and so on.
```
Comment (lukasjhan) [inline README.md:56]: About versioning

File: README.md (hunk 3 above)
Comment (cre8) [reply README.md:3]: To make it simple I suggest to sync the versions. Keeping them independant will increase the trouble on our side.

Maybe this PR is a good change tp add a reference in the contribution section.

File: packages/utils/README.md (hunk 5 above)
Comment (cre8) [reply packages/utils/README.md:1]: I would remove anything from the packages readme that does not belong there, e.g. the code coverage of the project.

Comment (lukasjhan) [reply packages/utils/README.md]: Okay :) done in [900158b](https://github.com/openwallet-foundation-labs/sd-jwt-js/pull/102/commits/900158b533c41259609522aec7169063788f3f11)

**Pull request merged**